### ISSUE TITLE
feat: `client.getUnsentMessages`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1611]
get an array of messages which have been not yet been sent (possibly due to connectivity issues)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.11.5](https://github.com/open-wa/wa-automate-nodejs/compare/3.11.4...3.11.5) (2021-04-26)

- build(deps): bump ts-json-schema-generator from 0.90.0 to 0.91.0 [`#1608`](https://github.com/open-wa/wa-automate-nodejs/pull/1608)
- build(deps): bump swagger-ui-dist from 3.46.0 to 3.47.1 [`#1604`](https://github.com/open-wa/wa-automate-nodejs/pull/1604)
- build(deps): bump pino from 6.11.2 to 6.11.3 [`#1603`](https://github.com/open-wa/wa-automate-nodejs/pull/1603)
- build(deps): bump puppeteer from 8.0.0 to 9.0.0 [`#1607`](https://github.com/open-wa/wa-automate-nodejs/pull/1607)
- build(deps): bump puppeteer-extra-plugin-stealth from 2.7.6 to 2.7.7 [`#1612`](https://github.com/open-wa/wa-automate-nodejs/pull/1612)
- build(deps-dev): bump eslint from 7.24.0 to 7.25.0 [`#1613`](https://github.com/open-wa/wa-automate-nodejs/pull/1613)
- build(deps-dev): bump typedoc from 0.20.35 to 0.20.36 [`#1609`](https://github.com/open-wa/wa-automate-nodejs/pull/1609)
- build(deps-dev): bump @types/node from 14.14.39 to 14.14.41 [`#1601`](https://github.com/open-wa/wa-automate-nodejs/pull/1601)
- Release 3.11.5 [`2c2673c`](https://github.com/open-wa/wa-automate-nodejs/commit/2c2673cc37bd60dae79655ad7d331759bf4400a6)
- ✨ feat [cli]: set `--config` via env var (WA_CLI_CONFIG) #1614 [`17d5486`](https://github.com/open-wa/wa-automate-nodejs/commit/17d5486352fc49cfca117422253a64bc4b2a4e9b)
- ✨ feat: `client.getUnsentMessages` #1611 [`1bddef9`](https://github.com/open-wa/wa-automate-nodejs/commit/1bddef967b4867be423f80a1438aed255be54408)
- 🥚 patches update [`158efc5`](https://github.com/open-wa/wa-automate-nodejs/commit/158efc59faf510b8e2babaddf8dcacb9d96f0a04)
- 📝 docs link README fix h/t maia [`4b705e4`](https://github.com/open-wa/wa-automate-nodejs/commit/4b705e43f859029f6131b954a534744b98115a24)
